docs/home.md

```diff
@@ -0,0 +1,42 @@
+# Design documentation for the Microsoft Authentication Library (MSAL) and Common library
+
+## Introduction
+
+This documentation provides orientation for new contributors to understand the design and design goals of the MSAL and the Common library supporting both MSAL and the Microsoft authentication broker libraries.  
+
+> NOTE: The Microsoft authentication broker library is currently private.
+
+## Android Libraries
+
+MSAL consists of 2 Android Libraries/Packages:
+
+- com.microsoft.identity.client.msal
+  - github: [https://github.com/AzureAD/microsoft-authentication-library-for-android](https://github.com/AzureAD/microsoft-authentication-library-for-android)
+- com.microsoft.identity.common
+  - github: [https://github.com/AzureAD/microsoft-authentication-library-common-for-android](https://github.com/AzureAD/microsoft-authentication-library-common-for-android)
+
+Where MSAL depends on common for definition of abstractions and implementation identity provider specific implementation.  
```

Comment:
This sentence seems to have a typo or mistake. May be re-write to:

`Where MSAL depends on common for definition of abstractions and identity provider specific implementation.`

Reply:
Fixed.